server/api/index.js

```diff
@@ -8,17 +8,60 @@ var express = require( 'express' );
  */
 var version = require( '../../package.json' ).version,
 	config = require( 'config' ),
+	request = require( 'request' ),
+	cookie = require( 'cookie' ),
+	cors = require( 'cors' ),
 	oauth = require( './oauth' );
 
+var corsOptions = {
+	origin: 'http://calypso.localhost:8008',
+	credentials: true
+};
+const TOKEN_NAME = 'wpcom_token';
+const URL = 'https://public-api.wordpress.com/rest/v1.1/sites/';
+const BLOG = 'alicoding.wordpress.com';
```

Comment:
Within the Page/Post editing interface on Calypso we know which WordPress instance we are on. It would be awesome if we can find a way to pass that information to the preview service so we can skip the manual config part.